bitmap-team-deuces/test/read-write-test.js

```diff
@@ -0,0 +1,18 @@
+'use strict';
+const fs = require('fs');
+const expect = require('chai').expect;
+const readwrite = require('../lib/readwrite');
+
+describe('Testing readwrite page', function(){
+  describe('Testing writeFile function', function(){
+    it('should write a new file called transformed.txt', function(done){
+      var testBuffer = new Buffer([1, 2, 3, 4, 5]);
+      var path = __dirname + '/../img/transformed.txt';
```

Comment:
I suggest you create all dummy data with regards to testing in a subfolder of the test folder which you delete at the end of your test(s).  It is important to delete it so you don't receive false positives when there are files from the previous tests still lying around :)